pyproject.toml

```diff
@@ -3,3 +3,31 @@ build-backend = 'setuptools.build_meta'
 # https://github.com/yt-dlp/yt-dlp/issues/5941
 # https://github.com/pypa/distutils/issues/17
 requires = ['setuptools > 50']
+
+[tool.ruff]
+exclude = [
+    "build",
+    "venv",
+    ".tox",
+    ".git",
+    ".pytest_cache",
+]
```

Comment:
Is there a way to include instead? SInce there's only a few folders it needs to lint.

Reply:
We can use `ruff check yt_dlp devscripts --respect-gitignore` instead. The current config was devised using `flake8-to-ruff`